cypress/e2e/pages/tokens/tokens-[symbol].spec.ts

```diff
@@ -253,19 +253,4 @@ context("/tokens/* Backing Addresses", () => {
         "https://www.blockchain.com/bch/address/38wFczGqaaGLRub2U7CWeWkMuPDwhMVMRf",
       );
   });
-
-  it("should redirect to individual token page", () => {
```

Comment:
why is this deleted? 🤔

Reply:
Its testing the same thing so I removed it